source/frontend/src/features/properties/map/project/update/UpdateProjectContainerView.tsx

```diff
@@ -0,0 +1,66 @@
+import { Input, Select, TextArea } from 'components/common/form';
+import { Section } from 'features/mapSideBar/tabs/Section';
+import { SectionField } from 'features/mapSideBar/tabs/SectionField';
+import { Formik, FormikProps } from 'formik';
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import styled from 'styled-components';
+
+import { AddProjectYupSchema } from '../add/AddProjectFileYupSchema';
+import { ProjectForm } from '../models';
+import { IUpdateProjectContainerViewProps } from './UpdateProjectContainer';
+
+const UpdateProjectContainerView = React.forwardRef<
+  FormikProps<ProjectForm>,
+  IUpdateProjectContainerViewProps
+>((props, formikRef) => {
+  const { initialValues, projectRegionOptions, projectStatusOptions, onSubmit } = props;
+
+  return (
+    <Formik<ProjectForm>
+      enableReinitialize
+      innerRef={props.formikRef}
+      initialValues={initialValues}
+      validationSchema={AddProjectYupSchema}
+      onSubmit={onSubmit}
+    >
+      {formikProps => (
+        <>
```

Comment:
nit: empty brackets not needed here given the container would count as the single child